Tavisca.Bootcamp.LanguageBasics.Exercise1/FixMultiplication.cs

```diff
@@ -0,0 +1,270 @@
+﻿using System;
+using System.Linq;
+
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
+{
+    class FixMultiplication
+    {
+        static void Main(string[] args)
+        {
+            Test("42*47=1?74", 9);
+            Test("4?*47=1974", 2);
+            Test("42*?7=1974", 4);
+            Test("42*?47=1974", -1);
+            Test("2*12?=247", -1);
+            Console.ReadKey(true);
+        }
+        
+        private static void Test(string args, int expected)
+        {
+           var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
+           Console.WriteLine($"{args} : {result}");
+        }
+
+        public static int FindDigit(string equation)
+        {
+
+            
+            int indexStar = 0;
+            int indexEqual = 0;
+            int indexQuestionMark = 0;
+
+            for (int k = 0; k < equation.Length; k++)
+            {
+                if(equation.ElementAt(k) == '*')
+                {
+                    indexStar = k;
+                }
+
+                if (equation.ElementAt(k) == '=')
+                {
+                    indexEqual = k;
+                }
+
+                if (equation.ElementAt(k) == '?')
+                {
+                    indexQuestionMark = k;
+                }
+            }
+
+            //finding the number with a missing digit
+            if (indexQuestionMark < indexStar && indexQuestionMark < indexEqual)  //number 1 is incomplete
+            {
+                // Number 1
+                int res = NumberOneIncomplete(indexStar, indexEqual, indexQuestionMark, equation);
+                return res;
+
+            }
+
+            else if (indexQuestionMark > indexStar && indexQuestionMark < indexEqual)  //number 2 is incomplete
+            {
+                // Number 2
+                int res = NumberTwoIncomplete(indexStar, indexEqual, indexQuestionMark, equation);
+                return res;
+            }
+
+            else if (indexQuestionMark > indexStar && indexQuestionMark > indexEqual)  //number 3 is incomplete
+            {
+                // Number 3
+                int res = NumberThreeIncomplete(indexStar, indexEqual, indexQuestionMark, equation);
+                return res;
+            }
+
+            throw new NotImplementedException();
+        }
+
+
+
+        public static int NumberOneIncomplete(int indexStar, int indexEqual, int indexQuestionMark, string equation)
+        {
+            //Number 1
```

Comment:
Local variables should also be in proper naming convention. So that the code is readable.